Content.Client/Arcade/BlockGameMenu.cs

```diff
@@ -282,16 +291,20 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
 
             _highscoresRootContainer.AddChild(menuInnerPanel);
 
-            var menuContainer = new VBoxContainer()
+            var menuContainer = new BoxContainer
             {
+            	Orientation = LayoutOrientation.Vertical,
```

Comment:
ok the indentation is just fucked on most of these so im just not gonna mention it further